packages/cf-deploy-config-writer/src/constants.ts

```diff
@@ -30,7 +30,7 @@ export const MbtPackageVersion = '^1.2.29';
 export const UI5TaskZipperPackage = 'ui5-task-zipper';
 export const UI5TaskZipperPackageVersion = '^3.4.x';
 export const UI5Package = '@ui5/cli';
-export const UI5PackageVersion = '^3.9.2';
+export const UI5PackageVersion = '^4.0.16';
```

Comment:
Out of curiosity, any reason for `^4.0.16` and `^4.0.0` in application-writer?

Reply:
I think I was just following the same existing syntax but I've now applied the base as 4.0.16, soo its consistent across all packages. Thanks.